scania-digital-design-system/sdds
# feat(slider) migrate Slider-component

**Describe pull-request**  
Migrated Slider component

**Solving issue**  
Fixes: #2416

**How to test**  
Check out the branch and click around.
Note that the eventlistener example will only work in canvas mode
